### ISSUE TITLE
Cannot expose built-in functions from the standard library

### ISSUE THREAD

**Issue by geoffbacon** [google/python-fire#187]
First of all, thanks for the awesome library! It makes writing CLIs fun...because you don't have to!

I couldn't find a similar existing issue nor did I come across anything on Google or StackOverflow. I'm trying to expose the functions from the [math module](https://docs.python.org/3/library/math.html) in the standard library, but this error replicates with other standard library modules with built-in functions (like [operator](https://docs.python.org/3/library/operator.html)). I have the following in script.py:

```python
import math
import fire

if __name__ == "__main__":
    fire.Fire(math)
```

I can access constants in math fine:
```
$ python script.py pi
3.141592653589793
```

but not the functions (which happen to be built-in):
```
$ python script.py sin 3
Traceback (most recent call last):
  File "script.py", line 6, in <module>
    fire.Fire(math)
  File "/Users/bacon/miniconda/envs/fire/lib/python3.7/site-packages/fire/core.py", line 138, in Fire
    component_trace = _Fire(component, args, parsed_flag_args, context, name)
  File "/Users/bacon/miniconda/envs/fire/lib/python3.7/site-packages/fire/core.py", line 471, in _Fire
    target=component.__name__)
  File "/Users/bacon/miniconda/envs/fire/lib/python3.7/site-packages/fire/core.py", line 675, in _CallAndUpdateTrace
    component = fn(*varargs, **kwargs)
TypeError: sin() takes exactly one argument (0 given)
```

Other standard library modules like calendar and unicodedata work fine too, because their functions aren't built-ins. A work-around is to wrap all the functions that you want in user-defined functions, but that would a lot of boilerplate (and lack of boilerplate is why fire > argparse/anything else).

I'm on Python 3.7.3 and fire 0.2.1.

**Comment by dbieber** [comment]
Thanks for flagging this issue.

Some builtins that are defined in C rather than in Python don't expose their signature to inspect.getargspec / inspect.getfullargspec. They may expose their signature to inspect.signature, in which case https://github.com/google/python-fire/issues/159 will resolve this issue for them. But for others, their signature isn't exposed to inspect at all. For these, I would like to introduce a module where we special case all of these functions. Unfortunately there's no comprehensive list of such builtins. So this is just going to be a best-effort fix. If you run into other modules with functions that Fire fails to handle correctly, you can add a comment w/ them on this bug and we'll try to handle those too.

**Comment by dbieber** [comment]
I had misdiagnosed the issue above. @geoffbacon found and resolved the actual issue in #193. 🙏
